### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by juniorjblp**
Added topic indicating the installation of react-native-cli for unexpected users

### CODE REVIEW

**Review by Simek — CHANGES_REQUESTED** [review]
Hi @juniorjblp, the global CLI installation is no longer required or recommended (you have also choose the legacy package):
* https://github.com/react-native-community/cli#about

Currently most of the command can be executed with `npx` (without installation):
* https://github.com/react-native-community/cli#creating-a-new-react-native-project

File: docs/getting-started.md [hunk 1]
```diff
@@ -133,6 +133,14 @@ You will need Node, the React Native command line interface, a JDK, and Android
 
 <block class="native windows android" />
 
+<h2> Intalling React Native Cli</h2>
+
+We recommend intalling Ract Native Cli
+
+```sh
+sudo npm install -g react-native-cli
```
Comment (Simek) [inline docs/getting-started.md:141]: You have added the section in the Android & Windows guide, without a WSL there is no Bash (and `sudo`) on Windows.

File: docs/getting-started.md [hunk 2]
```diff
@@ -478,7 +486,7 @@ Click "Next" then "Finish" to create your AVD. At this point you should be able
 
 <h3>Step 1: Start Metro</h3>
 
-First, you will need to start Metro, the JavaScript bundler that ships with React Native. Metro "takes in an entry file and various options, and returns a single JavaScript file that includes all your code and its dependencies."—[Metro Docs](https://facebook.github.io/metro/docs/concepts)
+First, you will need to start Metro, the JavaScript bundler that ships with React Native. Metro "takes in an entry file and various options, and returns a single JavaScript file that includes all your code and its ncies."—[Metro Docs](https://facebook.github.io/metro/docs/concepts)
```
Comment (Simek) [inline docs/getting-started.md:489]: This change is unnecessary.

**Comment by cpojer** [comment]
Closing this as I don't think we need this change?

**Pull request closed without merging**